kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -299,6 +299,50 @@ def exec_prove(
     sys.exit(failed)
 
 
+def exec_show_kcfg(
+    definition_dir: Path,
+    save_directory: Path,
+    spec_file: Path,
+    includes: List[str],
+    claim_label: Optional[str] = None,
+    spec_module: Optional[str] = None,
+    md_selector: Optional[str] = None,
```

Comment:
Nitpick: The highlighted part here is exactly the signature for exec_view_kcfg, except that the order of 'save_directory' and 'spec_file' is flipped. Let's re-order them for consistency then.

Reply:
Done.